src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/FileWatching/LspFileChangeWatcher.cs

```diff
@@ -64,7 +65,8 @@ private class FileChangeContext : IFileChangeContext
         /// The list of file paths we're watching manually that were outside the directories being watched. The count in this case counts
         /// the number of 
         /// </summary>
-        private readonly Dictionary<string, int> _watchedFiles = new Dictionary<string, int>(StringComparer.Ordinal);
+        private readonly Dictionary<string, int> _watchedFiles = new Dictionary<string, int>(_stringComparer);
```

Comment:
I wasn't sure exactly how to add a good test for this, am open to ideas + where it should go